static/gsAdmin/components/customers/updateRetentionSettingsModal.tsx

```diff
@@ -57,33 +60,32 @@ function UpdateRetentionSettingsModal({
 
   const onSubmit = () => {
     const retentions: Partial<
-      Record<DataCategory, {downsampled: number | null; standard: number}>
+      Record<DataCategory, {downsampled: number | null; standard: number | null}>
     > = {};
 
-    if (hasCategoryFeature(DataCategory.LOG_BYTE, subscription, organization)) {
```

Comment:
Check plan categories directly, because `hasCategoryFeature` also checks features, which is no the behavior we want here.